opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableDetails.tsx

```diff
@@ -0,0 +1,562 @@
+      window_title
+      startup_info {
+        key
+        value
+      }
+      integrity_level
+      ## windows-service-ext
+      service_name
+      descriptions
+      display_name
+      group_name
+      start_type
+      service_type
+      service_status
+    }
+    ... on Software {
+      observableName: name
+      cpe
+      swid
+      languages
+      vendor
+      version
+      x_opencti_product
+    }
+    ... on Url {
+      value
+    }
+    ... on UserAccount {
+      extensions
+      user_id
+      credential
+      account_login
+      account_type
+      display_name
+      is_service_account
+      is_privileged
+      can_escalate_privs
+      is_disabled
+      account_created
+      account_expires
+      credential_last_changed
+      account_first_login
+      account_last_login
+    }
+    ... on WindowsRegistryKey {
+      attribute_key
+      modified_time
+      number_of_subkeys
+    }
+    ... on WindowsRegistryValueType {
+      observableName: name
+      data
+      data_type
+    }
+    ... on Hostname {
+      value
+    }
+    ... on CryptographicKey {
+      value
+    }
+    ... on CryptocurrencyWallet {
+      value
+    }
+    ... on Text {
+      value
+    }
+    ... on UserAgent {
+      value
+    }
+    ... on BankAccount {
+      iban
+      bic
+      account_number
+    }
+    ... on Credential {
+      value
+    }
+    ... on TrackingNumber {
+      value
+    }
+    ... on PhoneNumber {
+      value
+    }
+    ... on PaymentCard {
+      card_number
+      expiration_date
+      cvv
+      holder_name
+    }
+    ... on MediaContent {
+      title
+      content
+      media_category
+      url
+      publication_date
+    }
+    ... on Persona {
+      persona_name
+      persona_type
+    }
+    ... on SSHKey {
+      key_type
+      public_key
+      fingerprint_sha256
+      fingerprint_md5
+      key_length
+      expiration_date
+      comment
+      created
+    }
+    ...StixCyberObservableIndicators_stixCyberObservable
+  }
+`;
+
+const reorderMediaContentObservablesAttributes = (data: { key: string, value: string }[]) => {
+  const desiredOrder = ['content', 'title', 'media_category', 'url', 'publication_date'];
+
+  return desiredOrder
+    .map((key) => data.find((item) => item.key === key))
+    .filter(Boolean) as { key: string, value: string }[];
+};
+
+const useTransformValue = (value: string | boolean | string[] | number, key: string) => {
+  const { dateAttributes } = useAttributes();
+  const { fldt } = useFormatter();
+
+  const result = includes(key, dateAttributes) ? fldt(value) : value;
+
+  if (result === true) return 'TRUE';
+  if (result === false) return 'FALSE';
+  if (Array.isArray(result)) return result.join('\n');
+  if (typeof result === 'number') return result.toString();
+
+  return result;
+};
+
+interface DownloadFileButtonMenuProps {
+  fileSize: number | null | undefined,
+  encodedFilePath: string | null,
+}
+
+const DownloadFileButtonMenu = ({
```

Comment:
Same here, maybe we can move it to a generic component folder ? To make the component less big